src/openrct2/ride/Vehicle.cpp

```diff
@@ -8906,53 +8906,56 @@ void Vehicle::UpdateTrackMotionMiniGolfVehicle(Ride* curRide, rct_ride_entry* ri
                 }
                 track_progress++;
                 break;
-            case 1: // loc_6DC7ED
+            case MiniGolfState::Unk1: // loc_6DC7ED
```

Comment:
This case is never set in the moveinfo so its safe to just assert false it. It also means the var_d3 is never set. It may be that RCT1 saves have a value in D3 though so removing it might cause some super minor difference.